config/proxy-config.js

```diff
@@ -42,9 +42,8 @@ module.exports = {
     assets: ['/files/press-kit.zip'],
   },
   waypoint: {
-    // actually https://waypointproject.io, but using wp.snarglepuss.com as a test
-    domain: 'https://wp.snarglepuss.com',
-    host: '(wp\\.snarglepuss\\.com|test-wp\\.hashi-mktg\\.com)',
+    domain: 'https://www.waypointproject.io',
+    host: '(www\\.waypointproject\\.io|test-wp\\.hashi-mktg\\.com)',
```

Comment:
waypointproject.io gets redirected to www.waypointproject.io in vercel settings, so only worrying about the www. subdomain in the app code.